config/tg-notifier.php

```diff
@@ -26,7 +26,7 @@
 
         'view' => [
             'path' => $_ENV['TGN_VIEW_PATH'] ??
-                'resources/views/telegram-git-notifier',
+                'resources/views',
```

Comment:
fix the path to use in the [app](https://github.com/lbiltech/telegram-git-notifier-app)